app/components/Tab/Tab.jsx

```diff
@@ -166,6 +163,10 @@ export default class Tab extends Component
                 this.loadURL( nextProps.url );
             }
         }
+
+        if (nextProps.pageIsLoading) {
```

Comment:
I think this is a mismatch of the props. 

If the page is loading, we should (eventually) show a loading indicator. (Which means 'page loading' probably needs to be set per tab, actually).

What we probably need here is a 'pageShouldReload' prop perhaps, on the ui store. That could even (possibly) be a string, 'done', 'soft', or 'hard', for reload type (maybe,... might be cleaner as a bool also).